ACMILabs/media-player
# #47 delete unneeded media files

*Resolves issue #47*

Delete unneeded media files if playlist is updated.

### Acceptance Criteria
- [x] Check if files on the file system match the playlist resources and delete any that don't
- [x] Download new files

### Relevant design files
* None

### Testing instructions
1. See restless-butterfly on Balena: https://dashboard.balena-cloud.com/devices/bd312d46b0571b07d7adf8cf7bae75d2/summary
1. Update the `PLAYLIST_ID` variable: https://dashboard.balena-cloud.com/devices/bd312d46b0571b07d7adf8cf7bae75d2/envvars
1. Watch the logs to see that the files from the old playlist are deleted. The log line should read something like: `Deleting unneeded media file: {file_to_delete}`

### DoD
For requester to complete:
